fastchat/conversation.py

```diff
@@ -306,10 +306,10 @@ def get_conv_template(name: str) -> Conversation:
     )
 )
 
-# Vicuna v1.1 template
+# Vicuna v1.3 template
 register_conv_template(
     Conversation(
-        name="vicuna_v1.1",
+        name="vicuna_v1.3",
```

Comment:
This update is not required. v1.1 and v1.3 use the same template, keeping the name preserved backward compatibility.